solidity/test/FixedRatePoolConverter.js

```diff
@@ -192,7 +192,7 @@ describe('FixedRatePoolConverter', () => {
     it('verifies the TokenRateUpdate event after removing liquidity', async () => {
         const converter = await initConverter(true, false);
 
-        const res = await converter.removeLiquidity(
+        const res = await converter.methods['removeLiquidity(uint256,address[2],uint256[2])'](
```

Comment:
See my comment below (etc.)

Reply:
@lbeder : This request is revoked, correct?

Reply:
Yap